test/e2e/test_cli.py

```diff
@@ -234,66 +231,6 @@ def test_excel_to_json_list(self) -> None:
         self.assertListEqual(output_actual, output_expected)
         out_file.unlink()
 
-    def test_excel_to_json_resources(self) -> None:
-        excel_file = Path("testdata/excel2json/excel2json_files/test-name (test_label)/resources.xlsx")
-        out_file = self.testdata_tmp / "_out_resources.json"
-        self._make_cli_call(f"dsp-tools excel2resources '{excel_file.absolute()}' {out_file.absolute()}")
-        with open(out_file, encoding="utf-8") as f:
-            output_actual = json.load(f)
-        with open("testdata/excel2json/resources-output-expected.json", encoding="utf-8") as f:
-            output_expected = json.load(f)
-        self.assertListEqual(output_actual, output_expected)
-        out_file.unlink()
-
-    def test_excel_to_json_properties(self) -> None:
-        excel_file = Path("testdata/excel2json/excel2json_files/test-name (test_label)/properties.xlsx")
-        out_file = self.testdata_tmp / "_out_properties.json"
-        self._make_cli_call(f"dsp-tools excel2properties '{excel_file.absolute()}' {out_file.absolute()}")
-        with open(out_file, encoding="utf-8") as f:
-            output_actual = json.load(f)
-        with open("testdata/excel2json/properties-output-expected.json", encoding="utf-8") as f:
-            output_expected = json.load(f)
-        self.assertListEqual(output_actual, output_expected)
-        out_file.unlink()
-
-    def test_id2iri(self) -> None:
-        xml_file = Path("testdata/id2iri/test-id2iri-data.xml")
-        mapping_file = Path("testdata/id2iri/test-id2iri-mapping.json")
-        self._make_cli_call(f"dsp-tools id2iri {xml_file.absolute()} {mapping_file.absolute()}")
-        out_file = get_most_recent_glob_match(self.cwd / "test-id2iri-data_replaced_*.xml")
-        with open(out_file, encoding="utf-8") as f:
-            output_actual = f.read()
-        out_file.unlink()
-        with open("testdata/id2iri/test-id2iri-output-expected.xml", encoding="utf-8") as f:
-            output_expected = f.read()
-        self.assertEqual(output_actual, output_expected)
-
-    def test_id2iri_remove_resources(self) -> None:
-        xml_file = Path("testdata/id2iri/remove-resources/data.xml")
-        mapping_file = Path("testdata/id2iri/remove-resources/mapping.json")
-        self._make_cli_call(f"dsp-tools id2iri --remove-resources {xml_file.absolute()} {mapping_file.absolute()}")
-        out_file = get_most_recent_glob_match(self.cwd / "data_replaced_*.xml")
-        with open(out_file, encoding="utf-8") as f:
-            output_actual = f.read()
-        out_file.unlink()
-        with open("testdata/id2iri/remove-resources/output-expected.xml", encoding="utf-8") as f:
-            output_expected = f.read()
-        self.assertEqual(output_actual, output_expected)
-
-    @pytest.mark.filterwarnings("ignore::UserWarning")
-    def test_excel2xml(self) -> None:
```

Comment:
This one is already tested in `test/unittests/test_excel2xml.py:test_excel2xml()`